features/support/env.rb

```diff
@@ -36,6 +36,10 @@ def initialize(a,b,c)
 
 transactional = non_transactional.map {|t| "~#{t}" }
 
+Before do
+  IndexProxyRuleWorker.stubs(:perform_later)
```

Comment:
Why? :thinking:

Reply:
@Martouta 
It is run after_commit, so all tests with `disable_transactional_fixtures` and involving Proxy (basically all, because you create an account + service + proxy) will trigger indexation of Proxy.

We wanted to use ActiveJob::TestHelper `perform_enqueued_jobs` but it does not have the `:only` options in Rails 4.2, we need to update to rails 5.x

Reply:
yeah, cucumber enables Sidekiq.inline! for all features, without this, all cucumber tests fails